spec/helpers/schema_helper_spec.rb

```diff
@@ -286,6 +291,11 @@
         data = valid_data.merge(carry_supplies: "Foo")
         expect(validate_against_form_response_schema(data).first).to include("carry_supplies")
       end
+
+      it "does not return a list of errors when carry_supplies has a nil value" do
+        data = valid_data.merge(carry_supplies: nil)
+        expect(validate_against_form_response_schema(data)).to be_empty
+      end
```

Comment:
Again here, shouldn't the test on line 290, "returns a list of errors when carry_supplies has an unexpected value" fail as well?